README.md

```diff
@@ -33,6 +33,25 @@ services:
       POSTGRES_USER: warpSQLUser
       POSTGRES_PASSWORD: warpSQLPass
 ```
+# Environment Variables
+
+Ensure that you have a `.env.sample` file same directory as the `docker-compose.yml` file.
+Open the `.env.sample` file in a text editor.
+Provide placeholders for the required environment variables in the `.env.sample` file. Do not include any actual sensitive information in this file. Here's an example of how the `.env.sample` file should look:
+
+```
+POSTGRES_USER=
+POSTGRES_PASSWORD=
+```
+Replace `POSTGRES_USER` and `POSTGRES_PASSWORD` with the appropriate variable names specific to your project.
```

Comment:
remove this line. No need to update .env.sample file.

User needs to create .env file from .env.sample and update the variable values in it
`cp .env.sample .env`